src/ex2/main.js

```diff
@@ -1,10 +1,93 @@
 // Implement the `Main` class here
 
+class Main {
+    init () {
+        const listToDo = document.querySelector("#listToDo");
+        const buttonClearAll = document.querySelector("#buttonClearAll");
+        const buttonAdd = document.querySelector("#buttonAdd");
+        const removeSVG = '<i class="fa fa-trash"></i>';
+        const lengthTodoList = document.querySelector(".lengthTodoList");
 
-const main = new Main();
+        lengthTodoList.textContent =0;
+        eventListener()
 
+        function eventListener(){
+            document.addEventListener('keydown', addEnter);
+            buttonAdd.addEventListener('click',addTodo );
+            buttonClearAll.addEventListener('click',deleteListToDo );
+        }   
+
+        function renderTodoList () {
+            listToDo.innerHTML=""
+            itemManager.todoList.forEach((todo) => {
+                const todoItem = createListTodoDiv(todo);
+                listToDo.appendChild(todoItem);
+            });
+            lengthTodoList.textContent = itemManager.todoList.length;
+            setIsActiveClearButton()
+        };
+
+        function deleteListToDo(){
+        if (confirm("Are you sure??") == true) {
```

Comment:
nice touch with the confirmation!
few notes:
- you can just write if(confirm("hello")) without == true, since this is already a value
- the item manager should have a deleteAll function to be use from outside, instead of you controlling it from main.js